Framework/Geometry/src/Objects/CSGObject.cpp

```diff
@@ -47,7 +47,7 @@ using namespace Mantid::Kernel;
 namespace {
 
 /// A shift to add/subtract to a point to test if it is an entry/exit point
-constexpr double VALID_INTERCEPT_POINT_SHIFT{2.5e-05};
+constexpr double VALID_INTERCEPT_POINT_SHIFT{2e-5};
```

Comment:
Is this necessary? The aim of the PR seems to be to move away from using this constant so wasn't clear why its value needs updating

Reply:
I will give the suggested three point solution a try